### ISSUE TITLE
Dependency injection

### ISSUE THREAD

**Issue by elifoster** [FTB-Gamepedia/Tilesheets#97]
As described on #96, we can remove uses of `global`:

For example 
```php
global $wgArticlePath;
``` 
becomes
```php
$mainConfig = MediaWikiServices::getInstance()->getMainConfig();
$wgArticlePath = $mainConfig->get('ArticlePath');
```

We can also remove uses of `wf...`: See https://phabricator.wikimedia.org/source/mediawiki/browse/master/docs/Injection.md

**Comment by elifoster** [comment]
This is now in most cases required, not optional.

**Comment by elifoster** [comment]
Resolved by #113.
